gammapy/irf/tests/test_effective_area.py

```diff
@@ -84,6 +84,11 @@ def test_to_table():
     assert hdu.header["TUNIT1"] == aeff.axes["energy_true"].unit
 
 
+@requires_data()
+def test_to_table_is_pointlike(aeff):
+    hdu = aeff.to_table_hdu()
+    assert(bool(table.meta.pop("is_pointlike", None)) == False)
```

Comment:
Here you don't test on `hdu`. 
You could simply test `assert "is_pointlike" in hdu.header`

Reply:
Also: assert is a statement, not a function. `assert condition`, not `assert(condition)`

Reply:
Sorry @registerrier , I've pushed before reading your suggestions. 
I've pushed again.